### PR TITLE
Lps 104416 9 4

### PR DESCRIPTION
**Pull request by slnn**
Rename and resent the changes 

https://github.com/brianchandotcom/liferay-portal/pull/93706#event-3726223277

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:sf

**Comment by dantewang** [comment]
Fetched

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java [hunk 1]
```diff
@@ -2921,20 +2921,12 @@ public LayoutPageTemplateStructureModel newLayoutPageTemplateStructureModel(
...
-					fragmentEntryLinkIds.add(
-						fragmentEntryLinkModel.getFragmentEntryLinkId());
-				}));
-
-		JSONObject jsonObject = layoutData.getLayoutDataJSONObject();
-
-		layoutPageTemplateStructureRelModel.setData(jsonObject.toString());
+		layoutPageTemplateStructureRelModel.setData(
+			StringUtil.replace(
+				_readFile("layoutPageTemplateStructureRel-data-content.json"),
```
Comment (dantewang) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java:2926]: @slnn AFAIK this method might be called multiple times. If so, please still move reading of the file content to the constructor.
On the other hand, the file name should be consistent. The DDMStructure JSON for basic document is named `ddm_structure_basic_document.json`. Your file should also be named as `model_name_what_file_is_for.json`. So I think `layout_page_template_structure_rel_data.json` should be a better name.

**Comment by dantewang** [comment]
@slnn Please make changes and resend.

**Comment by slnn** [comment]
Please see https://github.com/dantewang/liferay-portal/pull/1734

**Pull request closed without merging**